cmd/statusd/main.go

```diff
@@ -227,8 +228,8 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 	nodeConfig.Discovery = *discovery
 
 	// Even if standalone is true and discovery is disabled,
-	// it's possible to use bootnodes in NodeManager.PopulateStaticPeers().
-	// TODO(adam): research if we need NodeManager.PopulateStaticPeers() at all.
+	// it's possible to use bootnodes in NodeManager.PopulateBootNodes().
```

Comment:
What was the reason behind changing `StaticPeers` to `BootNodes`? These peers in the cluster config file are static peers actually. They are not regular bootnodes.

Reply:
Oh, seen them in the same context as our former static peers. Will rename them back.